packages/office-ui-fabric-react/src/components/pickers/BasePicker.tsx

```diff
@@ -81,6 +81,10 @@ export class BasePicker<T, P extends IBasePickerProps<T>> extends BaseComponent<
 
   @autobind
   public dismissSuggestions() {
+    // Select the first suggestion if one is available when user leaves.
```

Comment:
if you can add a change file that will help get it published. This looks like a patch.

To create a change file, run `npm run change` from the root. It will compare your current branch with origin/master. 

This may not be the right branch to compare with. For example if you have an `upstream` remote pointing to `OfficeDev`, you can tell the tooling to compare that branch by running:

`npm run change -- -b upstream/master`